src/main/resources/config-migration.json

```diff
@@ -146,5 +146,34 @@
         "value": "domination_awards.artefact_offers.legacy"
       }
     ]
+  },
+    {
+    "version": "0.8.0",
+    "changes": [
+      {
+        "type": "REMOVE",
+        "path": "domination_awards.artefact_offers.default_14_5_2022"
+      },
+      {
+        "type": "REMOVE",
+        "path": "domination_awards.artefact_offers.default_14_5_2022.tier1"
+      },
+      {
+        "type": "REMOVE",
+        "path": "domination_awards.artefact_offers.default_14_5_2022.tier2"
+      },
+      {
+        "type": "REMOVE",
+        "path": "domination_awards.artefact_offers.default_14_5_2022.tier3"
+      },
+      {
+        "type": "REMOVE",
+        "path": "domination_awards.artefact_offers.default_14_5_2022.tier4"
+      },
+      {
+        "type": "REMOVE",
+        "path": "domination_awards.artefact_offers.default_14_5_2022.tier5"
+      }
```

Comment:
It's not necessary to remove a config element when its gone from the enum afaik.

Reply:
Well recently, we did have this issue.....where deleted nodes were just.....hanging round

And this, as I recall, was the reason why you actually added the remove function.
Thats my recollection anyway, but of course if its not necessary now thats fine too as it would be less work.